arch/arm/core/aarch32/Kconfig

```diff
@@ -271,9 +273,11 @@ endchoice
 
 endmenu
 
-source "arch/arm/core/cortex_m/Kconfig"
-source "arch/arm/core/cortex_r/Kconfig"
+source "arch/arm/core/aarch32/cortex_m/Kconfig"
+source "arch/arm/core/aarch32/cortex_r/Kconfig"
+
+source "arch/arm/core/aarch32/cortex_m/mpu/Kconfig"
 
-source "arch/arm/core/cortex_m/mpu/Kconfig"
+source "arch/arm/core/aarch32/cortex_m/tz/Kconfig"
```

Comment:
rsource

Reply:
ACK. Fix coming in V3. Closing to declutter the story. Thanks.